<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcountManageWidget</class>
 <widget class="QWidget" name="AcountManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>账号信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>105</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>权限:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_role">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>管理员</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>收银员</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>密码:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_pwd">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>用户名:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_name">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>确认密码:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_r_pwd">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbnAdd">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbnModify">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbnOk">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_pwd</tabstop>
  <tabstop>lineEdit_r_pwd</tabstop>
  <tabstop>pbnAdd</tabstop>
  <tabstop>pbnModify</tabstop>
  <tabstop>pbnOk</tabstop>
  <tabstop>comboBox_role</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
